docs/rules/use-parameterized-jpa-query.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Use Parameterized JPA Query
+ruleId: UseParameterizedJPAQuery
+since: 3.18.0
+minJavaVersion: 1.1
+remediationCost: 10
+links:
+    - displayName: "How To Fix SQL Injection: JPA"
+      url: "https://software-security.sans.org/developer-how-to/fix-sql-injection-in-java-persistence-api-jpa"
+    
+description:
+    Rule description.
+tags: ["Security"]
+---
+
+# Use Parameterized JPA Query
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+JPA defines the query language JPQL (Java Persistence Query Language). A JPQL query string may be constructed by concatenating string literals with user defined expressions (e.g. variables, method invocations, user input, etc). Such a query string concatenation in JPQL may cause the same kind of vulnerability to injection attacks as a native SQL query string because the user input may be interpreted as JPQL code.  
```

Comment:
Please use the link to JPQL language reference: 
[JPQL](https://docs.oracle.com/html/E13946_04/ejb3_langref.html)